en-US/firefox-ios.xliff

```diff
@@ -2973,8 +2978,13 @@
         <note>Additional information about your Enhanced Tracking Protection</note>
       </trans-unit>
       <trans-unit id="Settings.TrackingProtection.ProtectionLevel.Footer" xml:space="preserve">
```

Comment:
This is removing the existing string but adding back exactly the same is the same (look at French xliff file, you will see the existing translation gets removed)